pkg/types/builder_test.go

```diff
@@ -394,6 +433,9 @@ func TestBuild(t *testing.T) {
 					t.Fatalf("Build(...): -want atProvider, +got atProvider: %s", diff)
 				}
 			}
+			if diff := cmp.Diff(tc.want.validationRules, g.ValidationRules, test.EquateErrors()); diff != "" {
```

Comment:
`test.EquateErrors()` is not needed here, probably also not needed above.

```suggestion
			if diff := cmp.Diff(tc.want.validationRules, g.ValidationRules); diff != "" {
```

Reply:
Thanks @lsviben 

I have removed `test.EquateErrors()` from the string comparisons.